generators/server/files.js

```diff
@@ -1316,20 +1316,22 @@ const baseServerFiles = {
         // Create Cucumber test files
         { file: 'package/cucumber/CucumberIT.java', renameTo: generator => `${generator.testDir}cucumber/CucumberIT.java` },
         {
-          file: 'package/cucumber/stepdefs/StepDefs.java',
+          file: 'package/cucumber/steps/StepDefs.java',
           renameTo: generator => `${generator.testDir}cucumber/stepdefs/StepDefs.java`,
         },
         {
           file: 'package/cucumber/CucumberTestContextConfiguration.java',
           renameTo: generator => `${generator.testDir}cucumber/CucumberTestContextConfiguration.java`,
         },
-        { file: '../features/gitkeep', noEjs: true },
       ],
     },
     {
       condition: generator => generator.cucumberTests,
       path: SERVER_TEST_RES_DIR,
-      templates: ['cucumber.properties'],
```

Comment:
In the upgrade process, you need to add reference for the files moved and deleted